cmake/modules/Findliboath.cmake

```diff
@@ -14,6 +14,10 @@ find_library(LIBOATH_LIBRARY NAMES oath liboath PATHS
   /usr/local/lib
   /usr/lib)
 
+if (NOT(LIBOATH_INCLUDE_DIR AND LIBOATH_LIBRARY))
```

Comment:
https://github.com/ceph/ceph/blob/master/CMakeLists.txt#L396 the `REQUIRED` argument and the `find_package_handle_standard_args ()` call at https://github.com/ceph/ceph/pull/22424/files#diff-7594d85288457bbba8e4f4dc88809548R24 are supposed to take care of this . what goes wrong is the inconsistency of `liboath` and `oath` in `find_package()` call and `find_package_handle_standard_args()` call.

i fixed this issue at #22430 in another way. could you please take a look at it? thanks!